sys-devel/gcc/gcc7-7.3.0_2018_05_01.recipe

```diff
@@ -142,6 +141,54 @@ sourceDir=$(pwd)
 relativeInstallDir="develop/tools$secondaryArchSubDir"
 installDir="$prefix/$relativeInstallDir"
 objectsDir=$(pwd)/../${portVersionedName}-obj
+gccLibDir=lib/gcc/$effectiveTargetMachineTriple/$gccVersion
+
+defineDebugInfoPackage gcc${secondaryArchSuffix} \
+	"$gccLibDir"/cc1 \
+	"$gccLibDir"/cc1plus \
+	"$gccLibDir"/collect2 \
+	"$gccLibDir"/lto1 \
+	"$gccLibDir"/lto-wrapper \
+	"$installDir/bin"/cpp \
+	"$installDir/bin"/g++ \
+	"$installDir/bin"/gcc \
+	"$installDir/bin"/gcc-ar \
+	"$installDir/bin"/gcc-nm \
+	"$installDir/bin"/gcc-ranlib \
+	"$installDir/bin"/gcov \
+	"$installDir/bin"/gcov-tool
+#	Do we need theese?
+#	"$gccLibDir"/*.a \
+#	"$developLibDir"/*.a
+
+# Is it even possible to define debuginfo package for subpackages?
+# This way not working.
+#defineDebugInfoPackage gcc${secondaryArchSuffix}_fortran \
+#	"$installDir/bin"/gfortran \
+#	"$gccLibDir"/f951 \
+#	"$gccLibDir"/libgfortran.a
+#
+#defineDebugInfoPackage gcc${secondaryArchSuffix}_syslibs \
+#	"$relativeLibDir"/libatomic.so.$libatomicLibVersion \
+#	"$relativeLibDir"/libgcc_s.so.$libgccSoVersion \
+#	"$relativeLibDir"/libgfortran.so.$libgfortranLibVersion \
```

Comment:
And this is what I would suggest for the syslibs subpackage:
```
defineDebugInfoPackage gcc${secondaryArchSuffix}_syslibs \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libatomic.so.$libatomicLibVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libgcc_s.so.$libgccSoVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libgfortran.so.$libgfortranLibVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libgomp.so.$libgompLibVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libquadmath.so.$libquadmathLibVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libssp.so.$libsspLibVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libstdc++.so.$libstdcxxLibVersion \
	"${libDir/gcc$secondaryArchSuffix/gcc${secondaryArchSuffix}_syslibs}"/libsupc++.so
```

Reply:
```sh
defineDebugInfoPackage gcc${secondaryArchSuffix}_syslibs \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libatomic.so.$libatomicLibVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libgcc_s.so.$libgccSoVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libgfortran.so.$libgfortranLibVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libgomp.so.$libgompLibVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libquadmath.so.$libquadmathLibVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libssp.so.$libsspLibVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libstdc++.so.$libstdcxxLibVersion \
	"$(getPackagePrefix syslibs)/$relativeLibDir"/libsupc++.so
```

:)
``